src/wasm.h

```diff
@@ -1441,6 +1441,8 @@ class Module {
   std::map<Name, Import*> importsMap;
   std::map<Name, Export*> exportsMap;
   std::map<Name, Function*> functionsMap;
+  // Types of functions which are declared but not defined.
+  std::map<Name, FunctionType*> externTypesMap;
```

Comment:
I don't think we even need this? This is an implementation detail of llvm/s2wasm, it shouldn't cause additional complexity in wasm.h for all modules generated in all methods. That is, can't llvm/s2wasm just emit a normal function type for this, and use it like a normal function type - why do we need this map?